### ISSUE TITLE
UnboundLocalError: local variable 'alab_idx' referenced before assignment

### ISSUE THREAD

**Issue by craicrai** [dkriegner/xrayutilities#76]
I used xrayutilities 1.5.2.
Please see below:

>Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
>Type 'copyright', 'credits' or 'license' for more information
>IPython 7.6.1 -- An enhanced Interactive Python. Type '?' for help.
>
>In [1]: from xrayutilities.materials.cif import CIFFile                                                                                                       
>
>In [2]: xu_cif = CIFFile("data/cif_files/beta-CsCl.cif")                                                                                                      
>
>UnboundLocalError                         Traceback (most recent call last)
><ipython-input-2-29f746584ebc> in <module>
>----> 1 xu_cif = CIFFile("data/cif_files/beta-CsCl.cif")
>
>~/inbox_work/xrd_analysis_workflow/venv/lib/python3.5/site-packages/xrayutilities/materials/cif.py in __init__(self, filestr, digits)
>    213         self._default_dataset = None
>    214         self.data = {}
>--> 215         self.Parse()
>    216 
>    217     def __del__(self):
>
>~/inbox_work/xrd_analysis_workflow/venv/lib/python3.5/site-packages/xrayutilities/materials/cif.py in Parse(self)
>    239                 self.fid.seek(fidpos)
>    240                 name = line[m.end():].strip()
>--> 241                 self.data[name] = CIFDataset(self.fid, name, self.digits)
>    242                 if self.data[name].has_atoms and not self._default_dataset:
>    243                     self._default_dataset = name
>
>~/inbox_work/xrd_analysis_workflow/venv/lib/python3.5/site-packages/xrayutilities/materials/cif.py in __init__(self, fid, name, digits)
>    301         if config.VERBOSITY >= config.INFO_ALL:
>    302             print('XU.material: parsing cif dataset %s' % self.name)
>--> 303         self.Parse(fid)
>    304         self.SymStruct()
>    305 
>
>~/inbox_work/xrd_analysis_workflow/venv/lib/python3.5/site-packages/xrayutilities/materials/cif.py in Parse(self, fid)
>    481                 asplit = line.split()
>    482                 try:
>--> 483                     atom = get_element(asplit[alab_idx])
>    484                     apos = (floatconv(asplit[ax_idx]),
>    485                             floatconv(asplit[ay_idx]),
>
>UnboundLocalError: local variable 'alab_idx' referenced before assignment

**Comment by dkriegner** [comment]
I am not sure I understand how this error can appear. `alab_idx` should have been defined before this code path can be entered.
Could you please upload (or send by email if you do not want to put it up here) the CIF file which caused this error?

**Comment by craicrai** [comment]
My apologies.  There is no error if I use a cif file downloaded from ICSD.

I have attached the cif file which caused the error, in any case.  
[beta-CsCl.cif.txt](https://github.com/dkriegner/xrayutilities/files/3358146/beta-CsCl.cif.txt)
(I appended '.txt' to the file name because github does not allow uploading the file ending in '.cif')
I generated this file with pymatgen.  I don't want to use cif files from ICSD because they are copyrighted and I have this project on github.

Also, I am not attaching here the cif file from ICSD (which worked without errors) because it is copyrighted.
